builder/googlecompute/config.go

```diff
@@ -443,16 +446,27 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 		}
 	}
 
+	// Set the default image architecture to X86_64. If the user has specified a value ensure it is uppercase.
+	if c.ImageArchitecture == "" {
+		c.ImageArchitecture = "X86_64"
+	} else {
```

Comment:
I believe we don't even need the `else` statement here, regardless what the value was it can't hurt to capitalise it even if it's not necessary.

Reply:
I've removed the else, to have it uppercase irregardless of input.